tests/rubinrepoman_test.py

```diff
@@ -4,17 +4,66 @@
 registry, getting the list of available tags, and filtering
 that down into a list of images to pull.
 """
-
 from __future__ import annotations
 
 from typing import List
 
 from cachemachine.rubinrepoman import RubinRepoMan
-from cachemachine.types import CachedDockerImage
+from cachemachine.types import CachedDockerImage, DockerImage
 
+from .conftest import mock_registry
 from .docker_mock import DockerMock
 
-HOST = "registry.hub.docker.com"
+repo = "registry.hub.docker.com/lsstsqre/sciplat-lab"
+expected_names = {
+    "prepuller_pulled_recommended": "prepuller_pulled_recommended",
+    "recommended": "Recommended (Release r21.0.0)",
+    "r22_0_0_rc1": "Release Candidate r22.0.0-rc1",
+    "r21_0_0": "Release r21.0.0",
+    "w_2021_22": "Weekly 2021_22",
+    "w_2021_21_c0021.003": "Weekly 2021_21_c0021.003",
```

Comment:
Don't we want this to say something like `Weekly 2021 week 22, cycle 21`? This seems less human-readable than I would expect.

Reply:
....mmmmmaybe, but Simon preferred "Weekly 2021_22" to "Weekly 22" (although I think a lot of the objection was that the year was missing).  I wanted to write a backwards-compatible implementation first, which implied (to me) keeping the underscores.  That and how to handle multiple images-with-the-same-tag are things that I think we need a real discussion about.

Reply:
I think it would be worth explicitly calling out the cycle even without the broader change to the display format, since it sounds like the cycle is pretty important for T&S and that way it's clear to people they have the right cycle.

Reply:
OK, I did the cycle callout, but not the broader change, which should wait on a discussion.